test/bin/stress_testing.sh

```diff
@@ -47,13 +48,15 @@ function reboot_and_wait {
     fi
     sleep 10
   done
+  sleep 10
```

Comment:
I guess this is allowing some time for microshift to come up after the boot. The RF tests also need to wait for this to happen, but they do it in a different way. There is a [keyword](https://github.com/openshift/microshift/blob/main/test/resources/microshift-process.resource#L26-L29) to check the `/readyz` endpoint in api server, [another](https://github.com/openshift/microshift/blob/main/test/resources/microshift-process.resource#L31-L34) one to check `/livez`, and this is [tried](https://github.com/openshift/microshift/blob/main/test/resources/microshift-process.resource#L39-L42) a few times.
This might be safer than a sleep, which could take more than 10s and then fail the oc wait command anyway.

Reply:
thanks, this is a great idea. I just added a check for `/livez` endpoint.